packages/fast-components-react-base/src/context-menu-item/context-menu-item.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from "react";
+import Foundation, { HandledProps } from "@microsoft/fast-components-foundation-react";
+import { get } from "lodash-es";
+import { ContextMenuItemClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+import {
+    ContextMenuItemHandledProps,
+    ContextMenuItemManagedClasses,
+    ContextMenuItemProps,
+    ContextMenuItemUnhandledProps,
+} from "./context-menu-item.props";
+
+export enum ContextMenuItemRole {
+    menuitem = "menuitem",
+    menuitemradio = "menuitemradio",
+    menuitemcheckbox = "menuitemcheckbox",

```

Comment:
As a discussion point, should we use our camelCase for the naming here or conform to the HTML all lowercase? Could this be for example menuItem = "menuitem"?

Reply:
The reason I used lower-case is becuase it's the literal value of the role. I'm using the key in a way where casing matters.

I can `toLowerCase` if we want to do that - but that's the reason behind it.